src/rpc/full_node_rpc_api.py

```diff
@@ -328,8 +328,14 @@ async def get_network_space(self, request: Dict) -> Optional[Dict]:
         weight_div_iters = delta_weight / delta_iters
         tips_adjustment_constant = 0.65
         network_space_constant = 2 ** 32  # 2^32
+        eligible_plots_filter_mult = (
+            2 ** self.service.constants["NUMBER_ZERO_BITS_CHALLENGE_SIG"]
```

Comment:
why is it 2 ** and not just the NUMBER_ZERO_BITS_CHALLENGE_SIG?

Reply:
The first 8 bits must be 0 for that plot to be eligible, which means 1/256 plots are eligible. So we need to multiple our estimate by 2**8 to get the actual space